src/server.js

```diff
@@ -21,81 +21,86 @@
 // SOFTWARE.
 
 /* eslint-env node */
-import {Plugin} from 'fusion-core';
+import {withMiddleware, memoize} from 'fusion-core';
 import Emitter from './emitter';
 
+export class GlobalEmitter extends Emitter {
+  constructor() {
+    super();
+    this.from = memoize(ctx => {
+      return new ScopedEmitter(ctx, this);
+    });
+  }
+  emit(type, payload, ctx) {
```

Comment:
Under what use cases would one use the 3rd arg here, and under what use cases would one use emitter.from? Is there a strong enough motivation to have both be part of the API?

Reply:
This is purely implementation detail, and not exposed